src/api/api.ts

```diff
@@ -1,57 +0,0 @@
-import axios, { AxiosRequestConfig } from "axios";
-import { onRequest, onRequestError, onRepoAccessRequest } from "./onRequest";
-
-export const BASE_URL = process.env.REACT_APP_BASE_URL;
-
-export const fetchData = axios.create({
-  baseURL: BASE_URL,
-  headers: {
-    "Content-type": "application/json",
-    Accept: "application/json",
-  },
-});
-
-export const postData = axios.create({
-  baseURL: BASE_URL,
-  headers: {
-    "Content-type": "text/plain;charset=utf-8",
-    Accept: "text/plain;charset=utf-8",
```

Comment:
This needs to be migrated over. 

TO BE used in case of sending commitHash in payload; `postData.post("/run", checkout)`

Reply:
This header is added in the action that needs it:
https://github.com/input-output-hk/dapps-certification-web/blob/288fc2954503c70ce27ef919d1dabf7a0b7ba51e/src/store/slices/testing.slice.ts#L84

Reply:
But, I'm adding a parameter in the fetch method so it can be included by passing an argument
(and it can be reused without repeating the header declaration)